some-art/ViewModel.swift

```diff
@@ -13,36 +13,33 @@ protocol ViewModelDelegate: AnyObject {
 }
 
 extension ViewController {
-    class ViewModel {
-        
+    final class ViewModel {
         weak var delegate: ViewModelDelegate?
         public var url: URL? {
             didSet {
                 delegate?.updateImage()
             }
         }
         
+        var randomType: String {
```

Comment:
❓ I moved these into the viewModel, but maybe there was another reason they were located outside of it?